### PR TITLE
Update makeHTTPCall function with different router path and request url

### LINKED ISSUE #61: test cases not getting key ids from request url
makeHTTPCall function should get a different router path and request URL.

For example:-
Router path string:- "/organisations/{organisation_id:[0-9]+}/core_values"
Request url string:- "/organisations/1/core_values"

### PR DESCRIPTION
**Pull request by PradipDKshirsagar**
Fixes #61

### CODE REVIEW

File: go-backend/service/user_http_test.go [hunk 1]
```diff
@@ -67,9 +69,9 @@ func (suite *UsersHandlerTestSuite) TestListUsersWhenDBFailure() {
 	suite.dbMock.AssertExpectations(suite.T())
 }
 
-func makeHTTPCall(method, path, body string, handlerFunc http.HandlerFunc) (recorder *httptest.ResponseRecorder) {
+func makeHTTPCall(method, path, requestUrl, body string, handlerFunc http.HandlerFunc) (recorder *httptest.ResponseRecorder) {
```
Comment (iamanujvrma) [inline go-backend/service/user_http_test.go:72]: how is path and requestURL different ?

Comment (iamanujvrma) [reply go-backend/service/user_http_test.go:72]: i got that, but whatever you are trying to request for eg: /users/1234

you can request on it as well as define the router on that too. That will work, right ?

Comment (PradipDKshirsagar) [reply go-backend/service/user_http_test.go:72]: Yes

Comment (iamanujvrma) [reply go-backend/service/user_http_test.go:72]: closing this PR as not needed.

Comment (iamanujvrma) [inline go-backend/service/user_http_test.go:72]: change `requestUrl` to `requestURL` 
URL is an abbreviation

File: go-backend/service/user_http_test.go [hunk 2]
```diff
@@ -67,9 +69,13 @@ func (suite *UsersHandlerTestSuite) TestListUsersWhenDBFailure() {
 	suite.dbMock.AssertExpectations(suite.T())
 }
 
-func makeHTTPCall(method, path, body string, handlerFunc http.HandlerFunc) (recorder *httptest.ResponseRecorder) {
+// Use path variable for configure our test router with given handler and requestURL variable for serve the http request
```
Comment (iamanujvrma) [inline go-backend/service/user_http_test.go:72]: Change the comment to:

path: is used to configure router path (eg: /users/{user_id})
requestURL: current request path (eg: /users/1)

File: go-backend/service/user_http_test.go (hunk 1 above)
Comment (anilmaurya) [reply go-backend/service/user_http_test.go]: @iamanujvrma I am facing this issue.  Are you going to close this PR or merge it ?

**Pull request merged**